packages/mediacenter/LibreELEC-settings/scripts/factory-reset

```diff
@@ -71,19 +75,15 @@ if [ -f /storage/.cache/reset_oe ] ; then
         tune2fs -U $uuid $part 2>&1 >/dev/null
       fi
     fi
```

Comment:
Okay.  I think I've fixed it.  Good call on moving the StartProgress into the if statement.  It did make more sense there.